src/v2/Apps/Fair/Routes/__tests__/ExhibitorFilterContext.jest.tsx

```diff
@@ -0,0 +1,55 @@
+import { mount } from "enzyme"
+import React from "react"
+import { act } from "react-dom/test-utils"
+import {
+  ExhibitorFilterContextProvider,
+  initialExhibitorFilterState,
+  useExhibitorsFilterContext,
+} from "../ExhibitorFilterContext"
+
+describe("ExhibitorFilterContext", () => {
+  let context: ReturnType<typeof useExhibitorsFilterContext>
+
+  const getWrapper = (props = {}) => {
+    return mount(
+      <ExhibitorFilterContextProvider {...props}>
+        <TestComponent />
+      </ExhibitorFilterContextProvider>
+    )
+  }
+
+  const TestComponent = () => {
+    context = useExhibitorsFilterContext()
+    return null
+  }
+
+  it("boots with default filters", async () => {
+    getWrapper()
+    expect(context.filters).toEqual(initialExhibitorFilterState)
+  })
+
+  describe("behaviors", () => {
+    it("#onFilterClick", () => {
+      const spy = jest.fn()
+      getWrapper({ onFilterClick: spy })
+      context.onFilterClick!(
+        "sort",
+        "FAETURED_DESC",
```

Comment:
**typo**: Looks like this should be `FEATURED_DESC`?

Reply:
Not sure how I missed that twice 😅  Thanks!